quickwit/Cargo.toml

```diff
@@ -198,7 +198,7 @@ rdkafka = { version = "0.33", default-features = false, features = [
 ] }
 regex = "1.10.0"
 regex-syntax = "0.8"
-reqwest = { version = "0.11", default-features = false, features = [
+reqwest = { version = "0.12", default-features = false, features = [
```

Comment:
OpenDAL uses `reqwest 0.12` (along with `http 1.0`, `hyper 1.0`) now. Maybe I can make more PRs to help migrating other crates?

Reply:
We're still waiting on `tonic` to move to `hyper 1.0`.

Reply:
Got it. Let's wait for https://github.com/hyperium/tonic/pull/1670

Reply:
Hi, @guilload maybe this PR is happy to go?